pages/common/[.md

```diff
@@ -0,0 +1,25 @@
+# test
```

Comment:
Oops, yeah @bl-ue! I've just noticed this. We should update this.
```suggestion
# [
```

Reply:
Issue is though, the linter doesn't like that, see below?